packages/main/src/RatingIndicator.js

```diff
@@ -244,6 +264,20 @@ class RatingIndicator extends UI5Element {
 			} else if (up && this.value < this.max) {
 				this.value = Math.round(this.value + 1);
 				this.fireEvent("change");
+			} else if (spaceOrEnter) {
+				const proposedValue = Math.round(this.value + 1);
+				this.value = proposedValue > this.max ? 0 : proposedValue;
+				this.fireEvent("change");
```

Comment:
The following snippet repeats and could be extracted below the conditions statement:
``` js
this.value = __some_assignmenrt_here;
this.fireEvent("change");
```

Reply:
The value assignment differs depending on the condition and cannot be extracted. Firing of the change event is now extracted.